src/index.ts

```diff
@@ -1,5 +1,5 @@
 // Core APIs
-export { SWRConfig, mutate, createCache } from './use-swr'
+export { SWRConfig, mutate, createCache, mutateCustomCache } from './use-swr'
```

Comment:
not sure if we want to expose the internalMutate here since `createCache` has already covered the similar usage

Reply:
Yeah, I think it's not the best way to support the use case. We might have to find out another solution like exporting a function that returns a key like `INFINITE_PREFIX + getFirstPageKey(getKey)` rather than exporting a mutate funciton.